### DISCUSSION TITLE
Chromium Framework by Puppeteer

### DISCUSSION THREAD

**Discussion by mohamad-elbohsaly** [git-lfs/git-lfs#5103] [Q&A] [answered]
In my Node.js project, I cannot perform a push because of Chromium Framework's size is > 100 MB although I added it through the git lfs command.

<img width="1317" alt="Screen Shot 2022-09-06 at 4 29 13 PM" src="https://user-images.githubusercontent.com/18421260/188647675-ea426d46-abd0-4e5b-84fd-b9ae76d905b9.png">

**Comment by MauDavalos10** [comment, marked as answer]
@mado-bohsali did you find a solution for this? For me it doesn't make sense that the path indicates "node_modules..." so when you ignore node_modules it should aso ignore (on .gitignore file) all the directories inside of it.

**Reply by mohamad-elbohsaly** [reply]
Thank you MauDavalos. Git LFS solved it.

**Reply by MauDavalos10** [reply]
@mado-bohsali  is not really a good practice to use LFS in this case, you should consider revert your changes until the first time you committed you 'node_modules' directory, and the add 'node_modules/' directory o your .gitignore file

**Reply by mohamad-elbohsaly** [reply]
`git lfs track "node_modules/puppeteer/.local-chromium/**"`

**Reply by mohamad-elbohsaly** [reply]
`git rm --cached path/to/large/file`
